presto-hive/src/test/java/com/facebook/presto/hive/TestHiveClientConfig.java

```diff
@@ -230,6 +233,7 @@ public void testExplicitPropertyMappings()
                 .setDomainCompactionThreshold(42)
                 .setWriterSortBufferSize(new DataSize(13, Unit.MEGABYTE))
                 .setForceLocalScheduling(true)
+                .setNodeSelectionStrategy(NodeSelectionStrategy.HARD_AFFINITY.name())
```

Comment:
static import `HARD_AFFINITY`, and remove `.name()` if changed signature according to previous comment.